src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {useState} from 'react';
+
+import './Header.css';
+
+
+function Header() {
+
+    const [a, setA] = useState(true);
```

Comment:
I think we have to remove that, as we were testing useState, and maybe you forgot to remove that